src/main/java/com/kk/strong/controller/UserController.java

```diff
@@ -65,4 +66,9 @@ public ResponseEntity<WorkoutSessionDto> addWorkoutSessionForUser(@PathVariable
         return ResponseEntity.created(URI.create(String.format("/users/%s/workout_sessions", userId)))
                 .body(userService.addWorkoutSessionForUser(userId, workoutSessionDto));
     }
+
+    @GetMapping("/rankings/{exerciseType}/{rankingSortingType}")
+    public ResponseEntity<List<ExerciseDto>> getExercisesForUserAndExerciseType(@PathVariable ExerciseType exerciseType, @PathVariable RankingSortingType rankingSortingType) {
+        return ResponseEntity.ok(userService.getLatestExercisesOfExerciseTypeForAllUsersWithSorting(exerciseType, rankingSortingType));
+    }
```

Comment:
Not sure if it's ok for the UserController to return ExerciseDtos. I feel like this method should not be in this controller, but fetching this data requires a top-down filtering from all users to specific exercises, so it suits the UserService